src/ui/suspense/frameCache.ts

```diff
@@ -1,28 +1,28 @@
 import { Frame, PauseId } from "@replayio/protocol";
 
+import { createUseGetValue } from "bvaughn-architecture-demo/src/suspense/createGenericCache";
 import {
-  createGenericCache,
-  createUseGetValue,
-} from "bvaughn-architecture-demo/src/suspense/createGenericCache";
+  getFramesAsync as getFramesAsyncNew,
+  getFramesIfCached as getFramesIfCachedNew,
+  getFramesSuspense as getFramesSuspenseNew,
+} from "bvaughn-architecture-demo/src/suspense/FrameCache";
 import { createFrame } from "devtools/client/debugger/src/client/create";
 import { PauseAndFrameId } from "devtools/client/debugger/src/reducers/pause";
 import { formatCallStackFrames } from "devtools/client/debugger/src/selectors/getCallStackFrames";
-import { Pause } from "protocol/thread/pause";
-import { assert } from "protocol/utils";
+import { replayClient } from "shared/client/ReplayClientContext";
```

Comment:
We shouldn't be importing and using `replayClient` directly. This value _should always_ come from context, or it will break things (e.g. our Playwright test proxy logic)

Reply:
This will be removed in the follow-up PR I mentioned above.

Reply:
I mostly split this into 2 PRs to make reviewing easier. I'll land them together.

Reply:
Cool!